src/com/example/mutations.cljs

```diff
@@ -16,10 +16,10 @@
 (defmutation delete-root-user [_]
   (action [{:keys [state] :as env}]
           (github-signout)
-          (swap! state assoc :root/user {:user/authenticated? false})))
+          (swap! state assoc-in [:component/id :main :main/user] {:user/authenticated? false})))
 
 (defn submit-form* [state quiz-id]
-  (let [user (:root/user state)
+  (let [user (get-in state [:component/id :main :main/user])
```

Comment:
changed where the user is in the db so I had to update the references to the user as well. Im not sure we want the user in this component state but we'll see.